### PR TITLE
test: Add unit tests for pep8 errors

### PR DESCRIPTION
**Pull request by Raine-Yang-UofT**
## Proposed Changes

Add unit tests for pep8 errors E115, E122, E125, E127, E129, E131 for PycodestyleChecker. These tests are used to cover untested branches of node_printer module.

...

<details>
<summary>Screenshots of your changes (if applicable)</summary>

</details>

## Type of Change

_(Write an `X` or a brief description next to the type or types that best describe your changes.)_

| Type                                                                                    | Applies? |
| --------------------------------------------------------------------------------------- | -------- |
| 🚨 _Breaking change_ (fix or feature that would cause existing functionality to change) |          |
| ✨ _New feature_ (non-breaking change that adds functionality)                          |          |
| 🐛 _Bug fix_ (non-breaking change that fixes an issue)                                  |          |
| ♻️ _Refactoring_ (internal change to codebase, without changing functionality)          |          |
| 🚦 _Test update_ (change that _only_ adds or modifies tests)                            |    X      |
| 📚 _Documentation update_ (change that _only_ updates documentation)                    |          |
| 📦 _Dependency update_ (change that updates a dependency)                               |          |
| 🔧 _Internal_ (change that _only_ affects developers or continuous integration)         |          |

## Checklist

_(Complete each of the following items for your pull request. Indicate that you have completed an item by changing the `[ ]` into a `[x]` in the raw text, or by clicking on the checkbox in the rendered description on GitHub.)_

Before opening your pull request:

- [x] I have performed a self-review of my changes.
  - Check that all changed files included in this pull request are intentional changes.
  - Check that all changes are relevant to the purpose of this pull request, as described above.
- [x] I have added tests for my changes, if applicable.
  - This is **required** for all bug fixes and new features.
- [x] I have updated the project documentation, if applicable.
  - This is **required** for new features.
- [x] I have updated the project Changelog (this is required for all changes).
- [x] If this is my first contribution, I have added myself to the list of contributors.

After opening your pull request:

- [x] I have verified that the pre-commit.ci checks have passed.
- [x] I have verified that the CI tests have passed.
- [x] I have reviewed the test coverage changes reported by Coveralls.
- [x] I have [requested a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from a project maintainer.

## Questions and Comments

_(Include any questions or comments you have regarding your changes.)_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ list only contains the Pylint checkers enabled by default in PythonTA.
 
 ### 🔧 Internal changes
 
-- Updated changelog and pull request template formats
+- Updated changelog and pull request git putemplate formats
```
Comment (quiz3) [inline CHANGELOG.md:47]: Was this an intentional change? It looks like there may be a typo on Line 47

File: tests/test_custom_checkers/test_pycodestyle_checker.py [hunk 2]
```diff
@@ -169,18 +185,122 @@ def test_no_error_e262(self) -> None:
...
-            ignore_position=True,
+            pylint.testutils.MessageTest(
+                msg_id="pep8-errors",
+                line=2,
+                args=("E115", "line 2, column 0: expected an indented block (comment)"),
+            )
         ):
             self.checker.process_module(mod)
 
-    def test_no_error_e265(self) -> None:
```
Comment (quiz3) [inline tests/test_custom_checkers/test_pycodestyle_checker.py:182]: @Raine-Yang-UofT, I think you may have forgotten to pull before the most recent commit you made. Your PR essentially deletes my PR, since my work had not been merged when you created your branch. 

I believe if you do 
```git checkout your-branch-name
git pull upstream master
*resolve merge conflicts*
git push origin your-branch-name
```
that should fix this problem.

Comment (quiz3) [reply tests/test_custom_checkers/test_pycodestyle_checker.py:182]: Alternatively, Professor @david-yz-liu may prefer to resolve the conflicts on his end? I would imagine it is better to resolve such conflicts before merging with `master` though.

**Comment by Raine-Yang-UofT** [comment]
@quiz3 Thanks for pointing out. I just fixed the typo in change log. That is probably caused by the merge conflict. Also, I double-checked that I didn't override any of your unit tests. Perhaps I moved some of them out of place when I was manually resolving the conflict, but none of them are deleted. I think all conflicts between our changes have been resolved.

File: CHANGELOG.md [hunk 3]
```diff
@@ -45,6 +45,7 @@ list only contains the Pylint checkers enabled by default in PythonTA.
 ### 🔧 Internal changes
 
 - Updated changelog and pull request template formats
+- Added unit tests for pep8 errors E115, E122, E125, E127, E129, E131 for `PycodestyleChecker`
```
Comment (david-yz-liu) [inline CHANGELOG.md:48]: To be consistent with the other PR, use "PEP8" instead of "pep8"

File: tests/test_custom_checkers/test_pycodestyle_checker.py [hunk 4]
```diff
@@ -149,6 +149,22 @@ def test_no_error_e228(self) -> None:
         with self.assertNoMessages():
             self.checker.process_module(mod)
 
+    def test_error_e265(self) -> None:
```
Comment (david-yz-liu) [inline tests/test_custom_checkers/test_pycodestyle_checker.py:152]: It looks like when resolving a merge conflict, you accidentally moved these two tests for E265. These tests shouldn't be moved, as this adds unnecessary changes to this diff. To fix this, look at their original location in the master branch and move them back to their correct position.

**Review by david-yz-liu — APPROVED** [review]

**Comment by david-yz-liu** [comment]
Good job, @Raine-Yang-UofT! 🎉
